content/geometry/General.adoc

```diff
@@ -4,19 +4,41 @@
 The unit used for object dimensions is meters.
 
 == Coordinate systems
-The local coordinate system is a right-handed coordinate system according to
-ISO 8855 with the axes pointing to the directions mentioned below. For a non-rotated
-coordinate system, the following applies:
+
+ASAM OpenMATERIAL uses the following types of coordinate systems:
+
+* World coordinate system, see <<sec-World-coordinate-system>>
+* Local coordinate system, see <<sec-Local-coordinate-system>>
+
```

Comment:
Is it correct that both coordinate systems have x/y/z axes? I doublechecked with OpenDRIVE, because I wanted to reuse the images and in OpenDRIVE the local coordinate system has axes u/v/z, see 
![image](https://github.com/user-attachments/assets/5ee15689-0818-4f6f-91bf-5321bfcb3096)

The following paragraph is based on the description in OpenSCENARIO, where the spec provides more detailed information. Please check whether it is correct.

Reply:
We are also using the terms "yaw, pitch, and roll", which is consistent with OpenSCENARIO, but not with OpenDRIVE, where "heading" is used instead of "yaw". Do you agree?